A6-client-retries.md

```diff
@@ -429,9 +432,12 @@ The retry policy is transmitted to the client through the service config mechani
         "maxAttempts": number,
 
         // The original RPC will be sent immediately, but the maxAttempts-1 
-        // subsequent hedged RPCs will be sent at intervals of every hedgingDelayMs. 
+        // subsequent hedged RPCs will be sent at intervals of every hedgingDelay. 
         // Set this to 0 to immediately send all maxAttempts RPCs.
```

Comment:
Should this say "Set this to '0s' to ..."?  Or is "0" OK?

What is the default value if it is unset?

Reply:
Agree that this should say `"0s"` instead of `0`.

The default should be `"0s"` if unset.  Let's document that.